.github/workflows/test.yml

```diff
@@ -55,6 +55,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        xfail: [
+          "windows-py310-pluggy",
+          "windows-py313",
+          "ubuntu-py310-pluggy",
+          "ubuntu-py310-freeze",
+          "ubuntu-py313",
+          "macos-py310",
+          "macos-py313"
+        ]
```

Comment:
Oh, I didn't mean that you should put this list into each matrix factor. But rework the matrix so that each entry has it as true or false. I see that it can be difficult, though.

Reply:
Actually, this doesn't even do that, but multiplies the number of jobs..
```console
Error when evaluating 'continue-on-error' for job 'build'. .github/workflows/test.yml (Line: 252, Col: 24): Error from function 'fromJSON': Unexpected symbol: 'windows-py310-pluggy'. Located at line 1 position 1 within JSON., .github/workflows/test.yml (Line: 252, Col: 24): Unexpected value ''
```

Reply:
The solution would be having
```yaml
matrix:
  xfail:
  - false
```
And moving the jobs from the list into includes with `xfail: true`.

Reply:
Oh I see. I will keep this as is for now, seems simpler, but we can evolve to use a separate true/false for each variant later if we find it better.

Reply:
Oh now I get it. The way I did it did not work either.